pmultiqc/modules/fragpipe/fragpipe.py

```
@@ -122,12 +158,107 @@ def get_data(self):
+        # Parse combined_peptide.tsv
+        if self.fragpipe_files.get("combined_peptide"):
+            try:
+                peptide_path = self.fragpipe_files["combined_peptide"][0]
+                peptide_df, combined_peptide_column_valid = combined_peptide_reader(peptide_path)
+                log.info("Combined peptide data loaded successfully.")
+
+                if peptide_df is not None and combined_peptide_column_valid:
+                    self.peptide_id_gain = cal_peptide_id_gain(peptide_df)
+                    log.info("peptide statics loaded successfully.")
+
+            except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)